src/Restaurant.js

```diff
@@ -1,15 +1,32 @@
-import React from "react";
+import React, {useState} from "react";
+import RestaurantButton from "./RestaurantButton";
+import Order from "./Order";
 
-const Restaurant = () => {
+
+
+
+
+
+
+const Restaurant = (props) => {
+  
+  const  [orders, setOrders] = useState(0);
   const pizzas = 0;
+  function orderOne() {
+    setOrders(orders+ 1);
+ 
+  }
+  
   return (
     <div>
       <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+        <ul>
+       
+        <Order orderType={"Pizzas"} />
+        <Order orderType={"Salads"} />
+        <Order orderType={"Chocolate cake"} />
```

Comment:
As a slight improvement, you could instead create an array of orders `const orders = ["Pizzas", "Salads", "Chocolate cake"]` and then `{ orders.map(order => <Order orderType={order} />) }` to render out various orders